dotCMS/src/main/java/com/dotcms/rest/LicenseResource.java

```diff
@@ -56,19 +56,16 @@ public Response getAll(@Context HttpServletRequest request, @PathParam("params")
             for ( Map<String, Object> lic : LicenseUtil.getLicenseRepoList() ) {
                 JSONObject obj = new JSONObject();
                 for ( Map.Entry<String, Object> entry : lic.entrySet() ) {
-
+                    String key = entry.getKey();
+                    String value = (entry.getValue()==null) ? "" : String.valueOf(entry.getValue());
+                    obj.put(key, value );
                     //Lets exclude some data we don' want/need to expose
-                    if ( entry.getKey().equals( "serverid" ) ) {
-                        obj.put( entry.getKey(), entry.getValue() != null ? LicenseUtil.getDisplayServerId( (String) lic.get( "serverId" ) ) : "" );
-                        obj.put( "fullserverid", entry.getValue() != null ? entry.getValue() : "" );
-                    } else if ( entry.getKey().equals( "serverId" ) || entry.getKey().equals( "license" ) ) {
-                        //Just ignore these fields
-                    } else if ( entry.getKey().equals( "id" ) ) {
-                        obj.put( entry.getKey(), entry.getKey() != null ? entry.getValue() : "" );
-                        obj.put( "idDisplay", entry.getValue() != null ? LicenseUtil.getDisplaySerial( (String) entry.getValue() ) : "" );
-                    } else {
-                        obj.put( entry.getKey(), entry.getKey() != null ? entry.getValue() : "" );
-                    }
+                    if ( "serverid".equals(key) ) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "serverid" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)